packages/devextreme/js/__internal/grids/grid_core/sticky_columns/const.ts

```diff
@@ -0,0 +1,12 @@
+export enum StickyPosition {
+  Left = 'left',
+  Right = 'right',
+}
+
+export const CLASSES = {
+  leftStickyColumn: 'dx-left-sticky-column',
+  rightStickyColumn: 'dx-right-sticky-column',
+  firstRightStickyColumn: 'dx-first-right-sticky-column',
+  lastLeftStickyColumn: 'dx-last-left-sticky-column',
+  stickyColumns: 'dx-sticky-columns',
```

Comment:
question
Why we need `leftStickyColumn`, `rightStickyColumn` and just `stickyColumns`? Isn't right... and left... enough?